apps/builder/app/shared/nano-states/nano-states.ts

```diff
@@ -326,6 +326,14 @@ export const useSetAuthPermit = (authPermit: AuthPermit) => {
   });
 };
 
+const authTokenStore = atom<string | undefined>(undefined);
+export const useAuthToken = () => useValue(authTokenStore);
+export const useSetAuthToken = (authToken: string | undefined) => {
```

Comment:
I think you can add the store to registerContainers and put it into a  /shared/nano-states so that it's automatically synced

Reply:
No need in sync, canvas has view access always